src/frontend/src/views/Upload/components/Metadata/components/StaticTable/index.tsx

```diff
@@ -147,44 +145,139 @@ export default function StaticTable({
         <MuiTable stickyHeader>
           <TableHead>
             <StyledTableRow>
-              <StyledTableCell>
-                <IdColumn>
-                  {SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.sampleId}
-                </IdColumn>
-              </StyledTableCell>
-              <StyledTableCell>
-                {SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.privateId}
-              </StyledTableCell>
-              <StyledTableCell>
-                {SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.publicId}
-              </StyledTableCell>
-              <StyledTableCell>
-                {SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.collectionDate}
-              </StyledTableCell>
-              <StyledTableCell>
-                {SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.collectionLocation}
-              </StyledTableCell>
-              <StyledTableCell>
-                {SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.sequencingDate}
-              </StyledTableCell>
-              <PrivateTableCell align="center">
-                {SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.keepPrivate}
-              </PrivateTableCell>
+              <StyledHeaderTooltip
+                title={
+                  <>
+                    <B>Sample Name (from FASTA):</B> ID of the sample, extracted
+                    from uploaded FASTA file(s).
+                  </>
+                }
+                placement="bottom"
+                sdsStyle="light"
+                arrow={true}
```

Comment:
for boolean props, you don't need to pass `{true}`. Just including the prop name implies a true value

```suggestion
                arrow
```